compose-builder/docker-compose-base.yml

```diff
@@ -221,7 +221,7 @@ services:
       - no-new-privileges:true
 
   rulesengine:
-    image: emqx/kuiper:${KUIPER_VERSION}
+    image: lfedge/ekuiper:${KUIPER_VERSION}
```

Comment:
interesting, the image is now under lfedge?

Reply:
This is because they are now part of LF Edge and no longer run by just EMQ.